### PR TITLE
Add ColumnCount property on page to split content

### PR DESCRIPTION
**Pull request by DamienSAVOYAT**
Add Column property on page to allow split content in 1, 2 or 3 columns.

### CODE REVIEW

File: OpenXMLSDK.Engine/Word/ReportEngine/Renders/PageExtensions.cs [hunk 1]
```diff
@@ -42,6 +42,28 @@ public static void Render(this Page page, Models.Document document, OpenXmlEleme
                 };
                 sectionProps.AppendChild(pageMargins);
             }
+
+            // If Columns are defined on the page we split page in columns
+            if (context.TryGetItem(page.ColumnNumberKey, out DoubleModel ColumnNumberKey))
```
Comment (HisCodeness) [inline OpenXMLSDK.Engine/Word/ReportEngine/Renders/PageExtensions.cs:47]: private properties must start with lower case

File: OpenXMLSDK.Engine/Word/ReportEngine/Renders/PageExtensions.cs [hunk 2]
```diff
@@ -42,6 +42,28 @@ public static void Render(this Page page, Models.Document document, OpenXmlEleme
...
+            {
+                // Try to convert double to ColumnCountValues : 1, 2 or 3
```
Comment (HisCodeness) [inline OpenXMLSDK.Engine/Word/ReportEngine/Renders/PageExtensions.cs:49]: You can merge this if statement with the one above

File: OpenXMLSDK.Engine/Word/ReportEngine/Renders/PageExtensions.cs (hunk 1 above)
Comment (DamienSAVOYAT) [reply OpenXMLSDK.Engine/Word/ReportEngine/Renders/PageExtensions.cs]: Ok, changed.

File: OpenXMLSDK.Engine/Word/ReportEngine/Renders/PageExtensions.cs (hunk 2 above)
Comment (DamienSAVOYAT) [reply OpenXMLSDK.Engine/Word/ReportEngine/Renders/PageExtensions.cs]: Ok, changed.

**Review by HisCodeness — APPROVED** [review]

**Pull request merged**